.appveyor.yml

```diff
@@ -10,17 +14,24 @@ cache:
 
 environment:
   matrix:
+    - nodejs_version: '10'
     - nodejs_version: '8'
     - nodejs_version: '6'
 
+matrix:
+  fast_finish: true
+
 install:
   - ps: Install-Product node $env:nodejs_version
   - set CI=true
   - yarn install
 
+build: off
+
 test_script:
   - node --version
   - yarn --version
   - yarn test
 
-build: off
+skip_commits:
```

Comment:
This is unnecessary. Adding `[skip ci]` to the commit message works. Please remove this.

Reply:
Copied this from another project. Upon rereading their [docs](https://www.appveyor.com/docs/how-to/filtering-commits/#skip-directive-in-commit-message), I think you're right that it's built-in. Wonder if it always was...